css/builder.css

```diff
@@ -618,6 +618,10 @@ nav>.btn {
   padding-left: 57px;
 }
 
+.form-control[readonly] {
+  background-color: inherit !important;
```

Comment:
please confirm why `!important` is required

Reply:
@squallstar this is because when we adding the disabled property the background color is changing, then i need to change the background to `inherit` color